### PR TITLE
r4

### PR DESCRIPTION
**Pull request by shinovon**

### CODE REVIEW

File: src/ui/Commands.java [hunk 1]
```diff
@@ -46,7 +46,8 @@ of this software and associated documentation files (the "Software"), to deal
 	
 	// Video page commands
 	static final Command watchCmd = new Command(Locale.s(CMD_Watch), Command.OK, 10);
-	static final Command downloadCmd = new Command(Locale.s(CMD_Download), Command.SCREEN, 9);
+	static final Command downloadCmd = new Command(Locale.s(CMD_Download), Command.SCREEN, 8);
+	static final Command showLinkCmd = new Command(Locale.s(CMD_ShowLink), Command.SCREEN, 6);
```
Comment (Feodor0090) [inline src/ui/Commands.java:50]: Is link showing more important than downloading? Now it has higher priority.

File: src/App.java [hunk 2]
```diff
@@ -129,6 +129,7 @@ public void schedule(Object o) {
 	public static int height;
 
 	public void startApp() {
+		AppUI.display(new Form("Loading"));
```
Comment (Feodor0090) [inline src/App.java:132]: What about to display current operation inside it like `nmania` does?

File: src/ui/VideoForm.java [hunk 3]
```diff
@@ -163,6 +170,14 @@ public void commandAction(Command c, Displayable d) {
 			App.download(video.getVideoId());
 			return;
 		}
+		if(c == showLinkCmd) {
+			TextBox t = new TextBox("", "", 64, TextField.URL);
+			t.setString("https://www.youtube.com/watch?v=" + video.getVideoId());
```
Comment (Feodor0090) [inline src/ui/VideoForm.java:175]: Header?
To pass the content into the box via constructor?

File: src/App.java [hunk 4]
```diff
@@ -42,7 +42,7 @@ of this software and associated documentation files (the "Software"), to deal
 
 public class App implements Constants {
 	
-	public static final String ver = "r3.2";
+	public static final String ver = "r3.3";
```
Comment (Feodor0090) [inline src/App.java:45]: According to best minecraft practices major version is locked to **1**.
According to best symdev practices, we have **4.3** as version but **3.3** as release. If all versions are releaseable (every unique version is a public release), is it better to synchronize them (and drop version/release splitting at all)?

File: src/App.java (hunk 2 above)
Comment (shinovon) [reply src/App.java:132]: That's a good idea

File: src/App.java (hunk 4 above)
Comment (shinovon) [reply src/App.java:45]: The first release version was 1.3.1, so...

File: src/ui/Commands.java (hunk 1 above)
Comment (shinovon) [reply src/ui/Commands.java:50]: Idk how it works, but it really seems randomly-sorted on nokia

**Pull request merged**